engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -1595,6 +1591,23 @@ final class SystemColor {
   ///
   ///   * [isSupported], which returns whether a specific color is supported.
   static bool get platformProvidesSystemColors => false;
+}
+
+/// A palette of system colors specified in the operating system for a given [brightness].
```

Comment:
Since know there are two static property 1 for light and 1 for dart. This doc should mention how developer can pick which one to use and maybe give some examples

Reply:
Example added